lib/tokenize.cpp

```diff
@@ -1425,6 +1426,8 @@ void Tokenizer::simplifyTypedef()
                             removed1.resize(idx);
                         }
                     }
+                    replStart->isSimplifiedTypedef(true);
```

Comment:
Will this mark all of the tokens? Because we need to know if the `*` was part of the typedef. I assume we would still want to warn about cases like this:

```cpp
typedef int A;
void f(A* x) { if (x == nullptr) {} }
```

Since this is fixable by adding a `const`.